src/global/variables/clauses/IntegerLiteral.java

```diff
@@ -1,26 +1,81 @@
 package global.variables.clauses;
 
+import global.tools.InvalidClauseException;
+import global.utils.RegexUtil;
 import global.variables.Clause;
 
 public class IntegerLiteral extends Clause {
-    int value;
+    int value = 0;
+    int upper;
+    int lower;
 
     public IntegerLiteral(int value) {
         super();
         this.value = value;
+        this.upper = value;
+        this.lower = value;
+
     }
 
     public IntegerLiteral(int value, String name) {
         super(name);
         this.value = value;
+        this.upper = value;
```

Comment:
I don't think we should set `upper` and `lower` at all if they are not set by the user - this applies to the previous constructor as well